recipe/meta.yaml

```diff
@@ -27,10 +28,13 @@ requirements:
     - cmake                   # [win]
     - ninja                   # [win]
     - objconv                 # [osx]
-    - llvm-openmp             # [(osx or linux) and USE_OPENMP == 1]
+    - llvm-openmp             # [USE_OPENMP == 1]
     # Following is needed because the Fortran compiler add an rpath to the BUILD_PREFIX
     - _openmp_mutex * *_llvm  # [linux and USE_OPENMP == 1]
+    # for showing CPU info of CI agent
+    - numpy *
```

Comment:
Let's not introduce cycles like this.

Reply:
Yes, that's a fair point. Any other concerns?